src/App.js

```diff
@@ -1,22 +1,31 @@
-import React from 'react';
-import styled from 'styled-components'
+import React, { useState, useEffect } from 'react';
 import GlobalStyle from './theme/GlobalStyle'
 import ThemeProvider from './theme/ThemeProvider'
+import { db } from './utility/firebase'
 
 // temp for testing
 import ProgressBar from './components/ProgressBar'
 import TimeDisplay from './components/TimeDisplay'
 
-const ExampleText = styled.div`
-  color: ${p => p.theme.colors.text}
-`
 
 function App() {
+  const [hackathons, setHackathons] = useState()
+
+  // example getting all hackathons data
+  useEffect(() => {
+    db.collection('Hackathons').get().then(querySnapshot => {
```

Comment:
Would it make sense to make a collection for each hackathon so we don't grab more info than we need?

Reply:
This is just an example call, we can defs just grab one hackathon's data.